package.json

```diff
@@ -46,7 +46,7 @@
     "@babel/preset-flow": "7.18.6",
     "@babel/preset-react": "7.18.6",
     "@rollup/plugin-replace": "^2.3.1",
-    "@stripe/connect-js": "2.0.1-beta.6",
+    "@stripe/connect-js": ">=2.0.1-beta.7",
```

Comment:
Forgot to bump `peerDependencies`

Reply:
Can we keep the specific package here, while having the >= package in `peerDependencies`?

Reply:
It helps make the dev experience in this repo consistent